src/test/java/com/google/api/codegen/testdata/discoveries/java/java_translate.v2.json.baseline

```diff
@@ -10,53 +10,54 @@
  *    On other build systems, you can add the jar files to your project from
  *    https://developers.google.com/resources/api-libraries/download/translate/v2/java
  */
-
-import com.google.api.client.auth.oauth2.Credential;
+import com.google.api.client.googleapis.auth.oauth2.GoogleCredential;
 import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
 import com.google.api.client.http.HttpTransport;
 import com.google.api.client.json.JsonFactory;
 import com.google.api.client.json.jackson2.JacksonFactory;
 import com.google.api.services.translate.Translate;
+import com.google.api.services.translate.Translate.Detections.List;
 import com.google.api.services.translate.model.DetectionsListResponse;
 import java.io.IOException;
 import java.security.GeneralSecurityException;
 import java.util.ArrayList;
-import java.util.Collections;
-import java.util.List;
-
 
 public class TranslateExample {
-  public static void main(String[] args) throws IOException, GeneralSecurityException {
-    Credential credential = getAuth();
-
-    HttpTransport httpTransport = GoogleNetHttpTransport.newTrustedTransport();
-    JsonFactory jsonFactory = JacksonFactory.getDefaultInstance();
-
-    Translate translateService = new Translate.Builder(httpTransport, jsonFactory, credential)
-      .setApplicationName("Google Cloud Platform Sample")
-      .build();
-
-
-    // TODO: Change placeholder below to desired parameter value for the `list` method:
-
+  public void run() throws IOException, GeneralSecurityException {
     // The text to detect
-    List<String> q = new ArrayList<String>();
+    java.util.List<String> q = new ArrayList<>(); // TODO: Update placeholder value.
```

Comment:
It was a trade-off, admittedly, I was aware of this possibility case and I do still prefer it
The obvious fix for this is to fully qualify the request type and not the list type, but I've changed it back anyway

#556